### PR TITLE
Add holiday header

### PR DESCRIPTION
**Pull request by neoape**

### CODE REVIEW

File: src/TimeGrid.js [hunk 1]
```diff
@@ -175,6 +175,10 @@ export default class TimeGrid extends Component {
     )
   }
 
+  getHolidays(events) {
+    return events.filter(({ viewType }) => viewType === 'bankHolidays')
```
Comment (aMoRRoMa) [inline src/TimeGrid.js]: Please move it form events to separate prop "holidays" for calendar

File: src/TimeGridHeader.js [hunk 2]
```diff
@@ -34,20 +36,34 @@ class TimeGridHeader extends React.Component {
...
   handleHeaderClick = (date, view, e) => {
     e.preventDefault()
     notify(this.props.onDrillDown, [date, view])
   }
 
+  renderHeader = ({
+    holiday,
+    components: {
+      header: HeaderComponent = Header,
+      holidayHeader: HolidayHeaderComponent,
```
Comment (aMoRRoMa) [inline src/TimeGridHeader.js]: I think better just replace header component

File: src/Month.js [hunk 3]
```diff
@@ -216,7 +215,7 @@ class MonthView extends React.Component {
     let { localizer, components } = this.props
     let first = row[0]
     let last = row[row.length - 1]
-    let HeaderComponent = components.header || Header
```
Comment (aMoRRoMa) [inline src/Month.js]: You should leave header from lib as default if header not came from outside

File: src/TimeGridHeader.js [hunk 4]
```diff
@@ -47,7 +47,8 @@ class TimeGridHeader extends React.Component {
       getDrilldownView,
       getNow,
       getters: { dayProp },
-      components: { header: HeaderComponent = Header },
```
Comment (aMoRRoMa) [inline src/TimeGridHeader.js]: You should leave header from lib as default if header not came from outside

**Review by aMoRRoMa — CHANGES_REQUESTED** [review]

File: src/Month.js (hunk 3 above)
Comment (neoape) [reply src/Month.js]: updated

File: src/TimeGridHeader.js (hunk 4 above)
Comment (neoape) [reply src/TimeGridHeader.js]: updated

**Review by aMoRRoMa — APPROVED** [review]

**Pull request merged**